strip-coverletter-docker.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -eu
+set -xv
+in=$(realpath $1)
+bname_in=${in##*/} # /foo/bar/baz.pdf = baz.pdf
+
+out=$2
+bname_out=${out##*/}
+
+mkdir -p vol
+cp $in vol/$bname_in
+
+sudo docker run \
```

Comment:
you shouldn't need `sudo`: add your user to the `docker` group. `elife.docker` already does this.